.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,20 @@
-<!-- please add an icon to the title of this PR (see https://sigs.k8s.io/cluster-api/CONTRIBUTING.md#contributing-a-patch), and delete this line and similar ones -->
-<!-- the icon will be either ⚠️ (:warning:, major or breaking changes), ✨ (:sparkles:, feature additions), 🐛 (:bug:, patch and bugfixes), 📖 (:book:, documentation or proposals), or 🌱 (:seedling:, minor or other) -->
+<!-- Thanks for sending a pull request! Here are some tips for you:
+
+    1. If this is your first time, please read our contributor guidelines: https://github.com/kubernetes-sigs/cluster-api/blob/main/CONTRIBUTING.md#contributing-a-patch and developer guide https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/book/src/developer/guide.md
+
+    2. Please label this pull request according to what area(s) you are addressing. For reference on on PR/issue labels: https://github.com/kubernetes-sigs/cluster-api/labels?q=area+
+
+    3. Please add an icon to the title of this PR (see https://sigs.k8s.io/cluster-api/CONTRIBUTING.md#contributing-a-patch), and delete this line and similar ones
+    the icon will be either ⚠️ (:warning:, major or breaking changes), ✨ (:sparkles:, feature additions), 🐛 (:bug:, patch and bugfixes), 📖 (:book:, documentation or proposals), or 🌱 (:seedling:, minor or other) 
+-->
 
 **What this PR does / why we need it**:
 
 **Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
 Fixes #
+
+#### What type of PR is this?
+<!--
+Area example:
```

Comment:
```suggestion
Example for adding an area label (see https://github.com/kubernetes-sigs/cluster-api/labels?q=area for the list of available labels):
```

Reply:
thanks for the call out. I forgot about this PR 😵‍💫

Reply:
Line 14 has this called out instead.